POEApi.Model/POEModel.cs

```diff
@@ -127,6 +127,22 @@ void instance_Throttled(object sender, ThottledEventArgs e)
 
         private ITransport GetTransport(string email, bool offline)
         {
+            int customRequestLimit;
+            if (Settings.UserSettings.Keys.Contains("ApiRequestLimit") &&
+                int.TryParse(Settings.UserSettings["ApiRequestLimit"], out customRequestLimit))
+            {
+                if (HttpTransport.AdjustThrottleWindowLimit(customRequestLimit))

```

Comment:
Can you change the message generation to a ternary please, more succinct.

Reply:
Something like this?